base/_typography.scss

```diff
@@ -238,11 +232,172 @@ table {
 	}
 
 	// zebra striping pattern
-	&.striped {
-		tbody {
-			tr:nth-of-type(odd) {
-				background-color: $table-border-color;
-			}
+	tbody {
+
+		tr:nth-of-type(odd) {
+			background-color: $table-border-color;
+		}
+	}
+}
+
+// Inline typography elements
+strong,
+b,
+em,
+i,
+cite,
+small,
+sup,
+sup { letter-spacing: .03125em; }
+
+sub,
+sup {
+	font-size: 75%;
+	line-height: 1;
+	position: relative;
+}
+
+sup { top: -0.5em; }
+
+sub { bottom: -0.25em; }
+
+abbr[title],
+cite,
+dfn[title] { border-bottom: 1px dotted palette(brand); }
+
+abbr[title],
+dfn[title] { cursor: help; }
+
+abbr[title] {
+	font-size: 75%;
+	font-weight: 700;
+	letter-spacing: 0.125em;
+	text-transform: uppercase;
+}
+
+b,
+strong { font-weight: 700; }
+
+i,
+em,
+cite { font-style: italic; }
+
+var {
+	color: palette(mono, 75);
+	display: inline-block;
+	font-style: italic;
+	padding: 0 .125em;
+}
+
+// Proper code blocks need to be surrounded by a <pre>. They work together.
+pre {
+	// scss-lint:disable ImportantRule
+	white-space: pre-wrap;
+	white-space: -moz-pre-wrap !important;	// 1999+ Mozilla
```

Comment:
Shouldn't need any prefix usage here, unless it has to do with the !important rule. I know that even with brunch we get Autoprefixer.